Automattic/wp-calypso
# Global Nav: Add collapsed global sidebar

Addresses issues https://github.com/Automattic/dotcom-forge/issues/6252 and https://github.com/Automattic/dotcom-forge/issues/6260

## Description

This PR will collapse the global sidebar when a site has been selected on the sites dashboard.

In follow up PRs, we will update the sites dashboard to show a site selector panel and a sites information panel on the select site action.

## Before

<img width="831" alt="Screenshot 2024-03-28 at 11 41 13" src="https://github.com/Automattic/wp-calypso/assets/5560595/8d8f69df-4b71-42f6-8721-4c229ca829b7">

## After

https://github.com/Automattic/wp-calypso/assets/5560595/7704089b-2658-4fe3-b1ce-a1cc2d6caa4b


## Testing Instructions
* This PR only applies to `/sites` dashboard. When you select a site by clicking on either the thumbnail or site name, the sidebar should collapse.
* Disable the feature flag `layout/dotcom-nav-redesign-v2` in development.json config file and `yarn start` calypso again. Check that the collapsed sidebar isn't visible and the global sidebar works as expected on all global nav pages.
